lib/datasource/aws-machine-image/readme.md

```diff
@@ -0,0 +1,117 @@
+> :warning: **This datasource is _experimental_**: Be aware that its syntax and behavior may change at any time!
+
+This datasource returns the latest [Amazon Machine Image](https://docs.aws.amazon.com/en_en/AWSEC2/latest/UserGuide/AMIs.html) via the AWS API (valid credentials required).
+
+Because there is no general lookupName, you have to use the [describe images filter](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-ec2/interfaces/describeimagescommandinput.html#filters) as minified JSON as a lookupName.
```

Comment:
```suggestion
Because there is no general `lookupName`, you have to use the [describe images filter](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-ec2/interfaces/describeimagescommandinput.html#filters) as minified JSON as a `lookupName`.
```

Do we want to use the monospaced font for `lookupName` maybe?